### DISCUSSION TITLE
Testnet Endurance node issues

### DISCUSSION THREAD

**Discussion by akumaigorodski** [ACINQ/eclair#1988] [Developers]
It does not get channels open, they just hang inoperational irregardless of number of confirmations, happens with both local Eclair and SBW. Guess issues is not an appropriate place for such stuff, hopefully this one is.

**Comment by pm47** [comment]
Hi, do you have a `node_id` or `channel_id` we can look at?

**Comment by akumaigorodski** [comment]
SBW nodeId: `028bafe2a5cd62a09aefab69d89364c34c706bff35a26d868585b141409ae19967`

Local Eclair nodeId: `0375920e80abe16647f9b9baad202bf2b7be6c796ca6f10a842c2cd248cda1585e`

**Reply by pm47** [reply]
Thanks for the report, should work now. Was a configuration error + a bug that kept us from detecting it. More on that later.

**Comment by t-bast** [comment]
Thanks for the report @btcontract!
There was a combination of:

- we didn't restart our bitcoin node to publish blockhash on ZMQ after applying #1910 
- the block timeout mechanism that was added in #1910 was actually buggy and is fixed in #1989: my bad, I owe you one!
